docs/source/configurations.rst

```diff
@@ -103,6 +105,7 @@ ARM Stratus
         name: dask-worker
         cores: 36
         memory: 270GB
+        shebang: '#!/usr/bin/env bash'
```

Comment:
You should remove the shebang from "real life" configurations, as keeping the default shebang is perfectly fine apparently for those. This include ARM, but also above Cheyenne and Nersc.

Reply:
Are these what the DOE are using as their `jobqueue.yaml` ? in that case they 'should' set the shebang k-v in this updated version.  Currently the config is set per-scheduler and the shebang is pulled from the scheduler's k-v config. This is solved if we use the deprecation method I suggested.  Although I think it's best practice for the documentation to show these as set so people understand.

Reply:
I think they shouldn't if they keep the default value, see my response to the other comment. I will try this tomorrow. Default shebang value should be inherited from default config if not specified.

Reply:
And so based on the above discussion, I think we really should remove the majority of examples containing `shebang` in the doc, only keeping a few specific. Maybe add the config you are using in your cluster in the example section?

Reply:
I deleted a bunch, I only left it in places where the docs describe the extra args as well!!